### PR TITLE
Make aETHc and rETH with tests

### LINKED ISSUE #182: Make aETHc and rETH with tests
Make an aETHc file where underlying of aETHc is calculated (with test file). Use function 'ratio' in the contract.
Make a rETH file where underlying of rETH is calculated (with test file). Use function 'Rethvalue' in the contract.

### PR DESCRIPTION
**Pull request by Richymaestro**
Fixes #182

### CODE REVIEW

File: defi_protocols/Ankr.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    eth_value_function: str = field(init=False) 
+    eth_value_abi: str = field(init=False)
+    
+    def __post_init__(self):
+        self.addr = Web3.to_checksum_address(self.addr)
+        self.blockchain = DERIVS_DB[self.addr]['blockchain']
+        if self.web3 is None:
+            self.web3 = get_node(self.blockchain, block=self.block)
+        else:
+            assert self.web3.isinstance(Web3), "web3 is not a Web3 instance"
```
Comment (sanekto) [inline defi_protocols/Ankr.py:39]: This kind of check is not "healthy". It is not needed that web3 is a Web3 instance. Just that the web3 object behaves like a Web3 object. So in general it is better not to check if a things is an instance of other thing if there is no specific need.

File: defi_protocols/Ankr.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        'blockchain': 'ethereum',
+        'underlying': '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+        'eth_value_function': 'sharesToBonds',
+        'eth_value_abi': '[{"inputs":[{"internalType":"uint256","name":"amount","type":"uint256"}],"name":"sharesToBonds","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"}]'
+    },
+}
+
+
+@dataclass
+class ETH_derivative:
```
Comment (sanekto) [inline defi_protocols/Ankr.py:24]: Class names should be CamelCase

**Comment by sanekto** [comment]
The code of `ETH_derivative` is exactly the same in both files, why not to declare it only once and import it from both modules?

File: defi_protocols/Ankr.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import logging
+
+from defi_protocols.util.EthDerivs import EthDerivative
+
+logger = logging.getLogger(__name__)
+
+def underlying(wallet, deriv_address, block, web3=None):
```
Comment (sanekto) [inline defi_protocols/Ankr.py:7]: this should receive the `decimals` argument right?

Comment (sgzerbo) [reply defi_protocols/Ankr.py:7]: You're right. Though these protocols have only been deployed on mainnet. But we should keep the blockchain argument nevertheless

**Comment by sgzerbo** [comment]
Since the code for both Rocket and Anker is exactly the same, due to the super cool ETHderivative class, perhaps we can include in the underlying of both protocols a sanity check to check that the address inputted belongs to the protocol addresses list. And raise an Exception "Incorrect address" or something like that if it doesn't

**Comment by sanekto** [comment]
Closed in favor of #183

**Pull request closed without merging**